frontend/src/components/admin/menu/DictionaryManagement.js

```diff
@@ -117,7 +117,31 @@ function DictionaryManagement() {
 
   const fetchedDictionaryMenu = (res) => {
     if (componentMounted.current) {
-      setDictionaryMenuz(res);
+      if (res) {
+        if (
+          res.toRecordCount !== undefined &&
+          res.fromRecordCount !== undefined &&
+          res.totalRecordCount !== undefined
+        ) {
+          setToRecordCount(res.fromRecordCount);
+          setFromRecordCount(res.toRecordCount);
+          setTotalRecordCount(res.totalRecordCount);
+        }
+        if (res.menuList) {
+          const menuList = res.menuList.map((item) => ({
+            id: item.id,
+            dictEntry: item.dictEntry,
+            localAbbreviation: item.localAbbreviation,
+            isActive: item.isActive,
+            categoryName: item.dictionaryCategory
+              ? item.dictionaryCategory.categoryName
+              : "not available",
+            lastupdated: item.lastupdated,
+          }));
+          console.log("new menu list: " + JSON.stringify(menuList));
```

Comment:
Let me hope you are removing this line. A screen shot would also suffice 👍 Great work done man.